flow-typed/npm/redux_v4.x.x.js

```diff
@@ -0,0 +1,59 @@
+// flow-typed signature: df80bdd535bfed9cf3223e077f3b4543
+// flow-typed version: c4c8963c9c/redux_v4.x.x/flow_>=v0.55.x
+
+declare module 'redux' {
```

Comment:
is it not possible to add these as an npm dependency? I don't think they should be part of redux-loop's src.

Reply:
There is a [central repository](https://github.com/flow-typed/flow-typed) where flow-types usually reside.

Reply:
Some points:

- I'm afraid this is not possible to add as npm. This is how Flow signatures work right now, either inline (it is get downloaded from flow-typed) or provided by the package
- this is not essential here, can be removed. I added it for testing
- even if it will be part of this repository, it will be more like developer dependency it will not be picked up by end-user's Flow

Reply:
i'm confused why flow-typed would not work? What does it do then? Seems like you could add a post-install script to your package to install the flow types for the packages you want

Reply:
done